recipes/obspy/meta.yaml

```diff
@@ -0,0 +1,65 @@
+# {% set version = "1.0.2.dev0" %}
+
+package:
+  name: obspy
+  version: {{ version }}
+
+source:
+#  fn: obspy-{{ version }}.zip
+#  url: https://pypi.python.org/packages/source/o/obspy/obspy-{{ version }}.zip
+  fn: condaforge_win.zip
+  url: https://github.com/megies/obspy/archive/condaforge_win.zip
```

Comment:
Don't worry, I just pointed the build to that branch's tip because I have put the skip condition mentioned above there. I just wanted to wait for feedback if running the tests in your CI is okay before merging that change into the current maintenance branch (for the next bug fix release).

If the build works on that branch we can switch the URL back to proper source distribution zipballs on pypi (commented lines right above), and we can start building the packages here starting with the next bug fix release.